src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RequestMappingService.java

```diff
@@ -146,11 +145,11 @@
         }
 
         //remove the entry of user from map in case of empty if suspended is false
-        log.info("Count of rejected access profiles in ORM : {} ", needToRemoveUAP.size());
-        log.info("Access profiles rejected by Drools in ORM: {} ", needToRemoveUAP);
+        log.info("Count of expired/suspended/rejected access profiles in ORM : {} ", needToRemoveUAP.size());
+        log.info("Access profiles for empty request for RAS: {} ", needToRemoveUAP);
 
         //remove the entry of user from map in case of empty if suspended is false
-        if (!needToRemoveUAP.isEmpty()) {
+        if (!needToRemoveUAP.isEmpty() && userType.equals(UserType.CASEWORKER)) {
```

Comment:
I think, it shouldn't be removed even for case-worker. we should remove this block.